### DISCUSSION TITLE
Property PermitAccessibilityExtractContent is missing in PdfSharp.Pdf.Security.PdfSecuritySettings class

### DISCUSSION THREAD

**Discussion by MikeGratsas** [empira/PDFsharp#203] [Q&A] [answered]
Previously the property PermitAccessibilityExtractContent to permit content extraction for accessibility was present in the PdfSharp.Pdf.Security.PdfSecuritySettings class in PDFsharp-1.5 version. However, it is missing in PDFsharp-6 version. Could you explain the reason?

**Comment by ThomasHoevel** [comment, marked as answer]
This bit is deprecated in the PDF 2.0 reference and is no longer supported by PDFsharp.

> Not used. This bit was previously used to determine whether content could be extracted for the purposes of accessibility, however, that restriction has been deprecated in PDF 2.0. PDF readers shall ignore this bit and PDF writers shall always set this bit to 1 to ensure compatibility with PDF readers following earlier specifications.
